ambry-api/src/main/java/com/github/ambry/config/RouterConfig.java

```diff
@@ -638,6 +638,9 @@ public class RouterConfig {
   @Config(ROUTER_UPDATE_OP_METADATA_RELIANCE_TIMESTAMP_IN_MS)
   public final long routerUpdateOpMetadataRelianceTimestampInMs;
 
+  // Group compression-related configs in the CompressConfig class.
+  private final CompressionConfig compressionConfig;
```

Comment:
you can set this to public, it's a readonly object. just like any other members of this RouterConfig class.

Reply:
Yes, I could make it public just to make it like others.

Reply:
please make it public.